src/qtiItem/helper/maxScore.js

```diff
@@ -121,9 +121,6 @@ export default {
                     maxScoreOutcome.buildIdentifier('MAXSCORE', false);
                 }
                 maxScoreOutcome.setDefaultValue(maxScore);
-            } else {
```

Comment:
we need to remove MAXSCORE if score of interaction cant be calculated, for example - it is infinity
`if (!hasInvalidInteraction || customOutcomes.size()) {`

Reply:
Thanks for this tip and explanations. I've created the check of the case when the processing is set to none and ExternalScored is `none`